client/src/components/SelectTabs/EventTypesTab.js

```diff
@@ -238,6 +238,7 @@ export default function EventTypesTab() {
         setUserURL(res.data);
       })
       .catch((err) => console.log(err));
+      console.log(userEvents)
   }, []);
```

Comment:
If you could go through and make sure there are no console warnings from your code after you get the changes from my merge before you merge that would be great!